params/paramstest/forks.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package paramstest
+
+import (
+	"github.com/ava-labs/avalanchego/upgrade/upgradetest"
+
+	"github.com/ava-labs/subnet-evm/params"
+)
+
+// ForkToChainConfig maps a fork to a chain config
+var ForkToChainConfig = map[upgradetest.Fork]*params.ChainConfig{
```

Comment:
This is already present in the `var` block of `vm_test.go.` I don't know if should be in a separate file here, or whether we can avoid the duplication altogether. At a gut level it seems silly to create a whole new file for a var block.